src/Microsoft.ML.AdditionalLearners/OlsLinearRegression.cs

```diff
@@ -466,7 +467,7 @@ public static void Pptri(Layout layout, UpLo uplo, int n, Double[] ap)
             Desc = "Train an OLS regression model.",
             UserName = UserNameValue,
             ShortName = ShortName,
-            XmlInclude = new[] { @"<include file='../Microsoft.ML.StandardLearners/Standard/doc.xml' path='doc/members/member[@name=""OLS""]/*' />" })]
+            XmlInclude = new[] { @"<include file='../Microsoft.ML.AdditionalLearners/Standard/doc.xml' path='doc/members/member[@name=""OLS""]/*' />" })]
```

Comment:
Is this the right path?

1. Why do we need to go up a directory, and then back into the same directory we are already in?
2. I don't see a `Standard` folder that `doc.xml` is in.  It looks like it is just directly under this current directory.

Reply:
The reference is relative to the CSharpApi.cs file, in Microsoft.ML. This line of xml comment will appear atop the OrdinaryLeastSqaresregressor in the C#api, and from there make its way in the XML. 

---
In reply to: [206638688](https://github.com/dotnet/machinelearning/pull/611#discussion_r206638688) [](ancestors = 206638688)